tests/test_units_operator.py

```diff
@@ -134,13 +142,6 @@ async def test_write_learning_params(units_operator: UnitsOperator):
     products = get_available_products(marketconfig.market_products, start)
     orderbook = await units_operator.formulate_bids(marketconfig, products)
 
-    units_operator.context.data_dict = {
-        "learning_output_agent_addr": "world",
-        "learning_output_agent_id": "export_agent_1",
-        "learning_agent_addr": "world_0",
-        "learning_agent_id": "learning_agent",
-    }
-
```

Comment:
it would be better to do it only here with
`units_operator.context.data.update({})` - if this is enough for you :)